webpage/github-pandoc.css

```diff
@@ -289,6 +302,25 @@ svg:not(:root) {
     overflow: hidden;
 }
 
+/**
+ * Hide hypothesis annotation toolbar, tooltip, etc. on print.
+ */
```

Comment:
Nice. It was annoying that printing would show the hypothesis toolbar. Does it also remove hypothesis highlights?

Reply:
No, I figured a user might want to retain those when printing? We should be easily able to remove that if you want though.

Reply:
> No, I figured a user might want to retain those when printing?

Agreed